jinad/api/endpoints/flow.py

```diff
@@ -73,24 +75,73 @@ def _create_from_pods(
     tags=[TAG]
 )
 def _create_from_yaml(
-    yamlspec: UploadFile = File(...)
+    yamlspec: UploadFile = File(...),
+    uses_files: List[UploadFile] = File([]),
+    pymodules_files: List[UploadFile] = File([])
 ):
     """ 
-    Build a flow using `Flow YAML` 
-    
-    > [Flow YAML Syntax](https://docs.jina.ai/chapters/yaml/yaml.html#flow-yaml-sytanx)
+    Build a flow using [Flow YAML](https://docs.jina.ai/chapters/yaml/yaml.html#flow-yaml-sytanx)
+
+    > Upload Flow yamlspec (`yamlspec`)
+
+    > Yamls that Pods use (`uses_files`) (Optional)
+
+    > Python modules (`pymodules_files`) that the Pods use (Optional)
+
+    **yamlspec**:
+
+        !Flow
+        pods:
+            encode1:
+                uses: test-if-encode1.yml
+            encode2:
+                uses: test-if-encode2.yml
+
+    **uses_files**: `test-if-encode1.yml`
+
+        !BaseTFEncoder
+        requests:
+        on:
+            IndexRequest:
+            - !EncodeDriver
+                if: doc.mime_type.startswith('text')
+
+    **uses_files**: `test-if-encode2.yml`
+
+        !BaseTFEncoder
+        requests:
+        on:
+            IndexRequest:
+            - !EncodeDriver
+                if: doc.mime_type.startswith('image')
+
 
     """
+
     with flow_store._session():
         try:
+            # This makes sure `uses` & `py_modules` are created locally in `cwd`
```

Comment:
should every `flow_store` create a file with its `id` as name to own their own files? So that there are no naming conflicts?